README.md

```diff
@@ -1,6 +1,70 @@
-# Setup #
+# Metro API #
 
-+  Run `docker-compose up`. 
-+  Open `http://localhost:3000` in your browser.
+## Overview ##
+This API allows you to manage resources for a metro managing company, including employees, trains, schedules, stations, tickets and maintenance.
 
-That's it.
+## Setup ##
+1. Run `docker-compose up`. 
+2. Open `http://localhost:3000` in your browser.
+
+## Endpoints API Documentation ##
+
+### Employees ###
+| Method | Endpoint | Description                               |
+| ------ | -------- |-------------------------------------------|
+| GET | /employees | Returns a list of all employees           |
+| GET | /employees/:id | Returns details about a specific employee |
+| POST | /employees | Creates a new employee                    |
```

Comment:
You should specify body and attack some exmples
Also please specify possible answers